### ISSUE TITLE
Use of non-latin characters for content title causes 500 server error

### ISSUE THREAD

**Issue by yukoff** [sulu/sulu#3367]
| Q | A
| --- | ---
| Bug? | yes
| New Feature? | no
| Sulu Version | 1.5+, dev-develop
| Browser Version | Chrome 58.0.3029.110, Avira Scout 17.5.3029.2702

#### Actual Behavior

Even without saving 500 error will be thrown for `post_node_resourcelocator_generate` route with the following backtrace:
```
[1] PHPCR\RepositoryException: Invalid path '/cmf/jamchernihiv/routes/uk/entertainment/'
    at n/a
        in /app/vendor/phpcr/phpcr-utils/src/PHPCR/Util/PathHelper.php line 329

    at PHPCR\Util\PathHelper::error('Invalid path \'/cmf/jamchernihiv/routes/uk/entertainment/\'', true)
        in /app/vendor/phpcr/phpcr-utils/src/PHPCR/Util/PathHelper.php line 55

    at PHPCR\Util\PathHelper::assertValidAbsolutePath('/cmf/jamchernihiv/routes/uk/entertainment/', false, true)
        in /app/vendor/phpcr/phpcr-utils/src/PHPCR/Util/PathHelper.php line 170

    at PHPCR\Util\PathHelper::normalizePath('/cmf/jamchernihiv/routes/uk/entertainment/')
        in /app/vendor/jackalope/jackalope/src/Jackalope/ObjectManager.php line 203

    at Jackalope\ObjectManager->getNodeByPath('/cmf/jamchernihiv/routes/uk/entertainment/')
        in /app/vendor/jackalope/jackalope/src/Jackalope/Session.php line 360

    at Jackalope\Session->nodeExists('/cmf/jamchernihiv/routes/uk/entertainment/')
        in /app/vendor/jackalope/jackalope/src/Jackalope/Node.php line 996

    at Jackalope\Node->hasNode('entertainment/')
        in /app/vendor/sulu/sulu/src/Sulu/Component/Content/Types/ResourceLocator/Mapper/PhpcrMapper.php line 389

    at Sulu\Component\Content\Types\ResourceLocator\Mapper\PhpcrMapper->isUnique(object(Node), '/entertainment/')
        in /app/vendor/sulu/sulu/src/Sulu/Component/Content/Types/ResourceLocator/Mapper/PhpcrMapper.php line 324

    at Sulu\Component\Content\Types\ResourceLocator\Mapper\PhpcrMapper->getUniquePath('/entertainment/', 'jamchernihiv', 'uk', null)
        in /app/vendor/sulu/sulu/src/Sulu/Component/Content/Types/ResourceLocator/Strategy/ResourceLocatorStrategy.php line 119

    at Sulu\Component\Content\Types\ResourceLocator\Strategy\ResourceLocatorStrategy->generate('Не-латиниця', 'b6ffab6a-2ddf-49f3-837f-7f006f49172b', 'jamchernihiv', 'uk', null)
        in /app/vendor/sulu/sulu/src/Sulu/Bundle/ContentBundle/Repository/ResourceLocatorRepository.php line 65

    at Sulu\Bundle\ContentBundle\Repository\ResourceLocatorRepository->generate(array('title' => 'Не-латиниця'), 'b6ffab6a-2ddf-49f3-837f-7f006f49172b', 'jamchernihiv', 'uk', 'default')
        in /app/vendor/sulu/sulu/src/Sulu/Bundle/ContentBundle/Controller/NodeResourcelocatorController.php line 45

    at Sulu\Bundle\ContentBundle\Controller\NodeResourcelocatorController->postGenerateAction(object(Request))
        in  line 

    at call_user_func_array(array(object(NodeResourcelocatorController), 'postGenerateAction'), array(object(Request)))
        in /app/var/cache/admin/dev/classes.php line 4228

    at Symfony\Component\HttpKernel\HttpKernel->handleRaw(object(Request), 1)
        in /app/var/cache/admin/dev/classes.php line 4183

    at Symfony\Component\HttpKernel\HttpKernel->handle(object(Request), 1, true)
        in /app/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php line 168

    at Symfony\Component\HttpKernel\Kernel->handle(object(Request))
        in /app/web/admin.php line 53
```

#### Expected Behavior

Normally, at this step, resourcelocator should be generated automatically.

#### Steps to Reproduce

Just create new content with title set to non-latin characters (eg. `"Не-латиниця"`)

**Comment by yukoff** [comment]
Seems this one is related to sulu/sulu-standard#698

**Comment by danrot** [comment]
Finally fixed by #5057
